docs/src/main/asciidoc/vertx-reference.adoc

```diff
@@ -0,0 +1,942 @@
+5. Once the reply is received by the sender, the content is written to the HTTP response
+
+
+=== Bi-directional communication with browsers using SockJS
+
+The SockJS bridge provided by Vert.x allows browser applications and Quarkus applications to communicate using the event bus.
+It connects both sides.
+So, both sides can send messages received on the other side.
+It supports the three delivery mechanisms.
+
+SockJS negotiates the communication channel between the Quarkus application and the browser.
+If WebSockets are supported, it uses them; otherwise, it degrades to SSE, long polling, etc.
+
+So use SockJS, you need to configure the bridge, especially the addresses that will be used to communicate:
+
+[source, java]
+----
+package org.acme.vertx;
+
+import io.vertx.core.Vertx;
+import io.vertx.ext.bridge.PermittedOptions;
+import io.vertx.ext.web.Router;
+import io.vertx.ext.web.handler.sockjs.SockJSBridgeOptions;
+import io.vertx.ext.web.handler.sockjs.SockJSHandler;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.event.Observes;
+import javax.inject.Inject;
+import java.util.concurrent.atomic.AtomicInteger;
+
+@ApplicationScoped
+public class SockJsExample {
+
+    @Inject
+    Vertx vertx;
+
+    public void init(@Observes Router router) {
+        SockJSHandler sockJSHandler = SockJSHandler.create(vertx);
+        sockJSHandler.bridge(new SockJSBridgeOptions()
+                .addOutboundPermitted(new PermittedOptions().setAddress("ticks")));
+        router.route("/eventbus/*").handler(sockJSHandler);
+    }
+
+}
+----
+
+This code configures the SockJS bridge to send all the messages targeting the `ticks` address to the connected browsers.
+More detailled explanations about the configuration can be found on https://vertx.io/docs/vertx-web/java/#_sockjs_event_bus_bridge[the Vert.x SockJS Bridge documentation].
+
+The browser must use the `vertx-eventbus` JavaScript library to consume the message:
+
+[source, html]
+----
+<!doctype html>
+<html>
+<head>
+    <meta charset="utf-8"/>
+    <title>SockJS example - Quarkus</title>
+    <script src="https://code.jquery.com/jquery-3.3.1.min.js"
+            integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
+    <script type="application/javascript" src="https://cdn.jsdelivr.net/sockjs/0.3.4/sockjs.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/vertx3-eventbus-client@3.8.5/vertx-eventbus.min.js"></script>
+</head>
+<body>
+
+<h1>SockJS Examples</h1>
+
+<p><strong>Last Tick:</strong> <span id="tick"></span></p>
+
+</body>
+<script>
+    var eb = new EventBus('/eventbus');
+
+    eb.onopen = function () {
+
+        eb.registerHandler('ticks', function (error, message) {
+            $("#tick").html(message.body);
+        });
+    }
+
+</script>
+</html>
+----
+
+[#native-transport]
+== Native Transport
+
+IMPORTANT: Native transports are not supported in native executables.
```

Comment:
```suggestion
IMPORTANT: Native transports are not supported in GraalVM produced binaries.
```

The idea is to avoid using 'native' that means two different things here